### ISSUE TITLE
STORM eller dele deraf?

### ISSUE THREAD

**Issue by ClockEnd** [JosefAssad/SeMaWi#44]
Tænker at skele til STORM og få indarbejdet enkelte klassifikationsnøgler herfra i SEMaWi....
Emne til diskussion

**Comment by ClockEnd** [comment]
Gentager lige...når vi er på klassifikationsvejen....

**Comment by JosefAssad** [comment]
Ja, det er en ret nice ide. Det kan vi bruge til at klassificere systemerne, korrekt?

Data ligger i et [regneark](http://digi.it-kartellet.dk/sites/default/files/STORM_2.1.2.xls) så det er ikke den HELT store opgave at indlæse.

Kan være det er noget vi skal snakke med DIGST om?

**Comment by JosefAssad** [comment]
@mariushart siger at referencedata.dk som kommer til at SMW vil være en god datakilde for STORM. Kan være vi skal vente til den er op og køre.

**Comment by JosefAssad** [comment]
Da vi venter på dataleverance fra referenacedata.dk bruger vi bulk import i første omgang fra regnearket.

**Comment by mariushart** [comment]
Jeg foreslår at vi afventer modelleringsarbejdet med LoRA og Klassifikation bl.a STORM før vi anvender STORM i wiki. STORM kræver at vi laver en nøjere gennemgang sammen for at vurdere den bedste anvendelse

**Comment by JosefAssad** [comment]
Indtil vi finder et godt standard framework eller beslutter andet må denne issue kobles fra en mmilestone.

I må gerne kigge på STORMs indhold (link i kommentar https://github.com/JosefAssad/SeMaWi/issues/44#issuecomment-161655071), jeg tror som minimum der mangler en ikke-triviel del lokal klassifikation oven på STORM for at det giver konkret værdi (happy to be corrected).

**Comment by mariushart** [comment]
Roger, jeg har netop i dag ledt efter en lokal klassifikation med udgangspunkt i ITIL - men foreløbigt uden positivt resultat.
